src/atc/delta/delta_handle.py

```diff
@@ -110,6 +112,35 @@ def recreate_hive_table(self):
         self.drop()
         self.create_hive_table()
 
+    def get_partitioning(self):
```

Comment:
what if there is no partion? Is that even possible?

Reply:
if it is - how does the method handle it?

Reply:
its possible, returns empty list. I added a test.